src/docker/workflows/base.ts

```diff
@@ -44,7 +44,7 @@ export abstract class LocalResourceWorkflow {
    */
   static imageRepo: string;
   /** Tag for the Docker image to use for this workflow. Should be configurable by the user in extension settings. */
-  imageTag: string = "latest";
+  imageTag: string = "7.9.0";
```

Comment:
Since this is the abstract base class, let's leave it at `latest`, but specifically set `imageTag` in the `ConfluentLocalWorkflow` and `ConfluentPlatformSchemaRegistryWorkflow` subclasses instead from the sections linked in https://github.com/confluentinc/vscode/pull/2410#discussion_r2302764349. (When we're able to support the Apache Kafka image(s) and Medusa/datagen, those won't use `7.9.0`.)